frontend/src/components/activePage/Toolbox/Toolbox.tsx

```diff
@@ -0,0 +1,257 @@
+      >
+        Destroy Selected{` (${selectiveDestroy.length})`}
+      </Button>
+      <Button
+        type={!selectiveDestroy.length ? 'primary' : 'warning'}
+        ghost
+        shape="round"
+        size="middle"
+        onClick={() => deselectAll()}
+        disabled={!selectiveDestroy.length}
+      >
+        Deselect All
+      </Button>
+    </div>
+  );
+  return (
+    <div className="h-full flex justify-start items-center gap-2 px-5 py-3">
+      <Search
+        allowClear
+        className="hidden sm:block"
+        placeholder="Search User"
+        onChange={event => setSearchField(event.target.value)}
+        onSearch={value => setSearchField(value)}
+        enterButton
+      />
+      <Popover
+        className="block sm:hidden"
+        content={
+          <div className="flex justify-center gap-4">
+            <Search
+              placeholder="Search User"
+              onChange={event => setSearchField(event.target.value)}
+              onSearch={value => setSearchField(value)}
+              allowClear
+              enterButton
+            />
+          </div>
+        }
+        placement="bottomRight"
+        trigger="click"
+        visible={searchPopover}
+        onVisibleChange={() => setSearchPopover(old => !old)}
+      >
+        <Button
+          type={searchPopover ? 'danger' : 'primary'}
+          shape="circle"
+          size="middle"
+          icon={!searchPopover ? <SearchOutlined /> : <CloseOutlined />}
+        />
+      </Popover>
+      <Divider type="vertical" className="hidden md:block" />
+      <Button
+        className="hidden md:block"
+        type="primary"
+        shape="round"
+        size="middle"
+        icon={<FullscreenOutlined />}
+        onClick={() => setExpandAll(true)}
+      >
+        Expand
+      </Button>
+      <Button
+        className="hidden md:block"
+        type="primary"
+        shape="round"
+        size="middle"
+        icon={<FullscreenExitOutlined />}
+        onClick={() => setCollapseAll(true)}
+      >
+        Collapse
+      </Button>
+      {/* <Tooltip title="Expand All" placement="top">
+        <Button
+          className="sm:hidden"
+          type="primary"
+          shape="circle"
+          size="middle"
+          icon={<FullscreenOutlined />}
+          onClick={() => setExpandAll(true)}
+        />
+      </Tooltip>
+      <Tooltip title="Collapse All" placement="top">
+        <Button
+          className="sm:hidden"
+          type="dark"
+          shape="circle"
+          size="middle"
+          icon={<FullscreenExitOutlined />}
+          onClick={() => setCollapseAll(true)}
+        />
+      </Tooltip> */}
+      <Divider type="vertical" className="hidden xl:block" />
+      <div className="flex flex-col justify-start gap-2 hidden xl:block">
+        <div className="flex items-center w-36 gap-2">
+          <Switch
+            checked={showAdvanced}
+            onClick={e => setShowAdvanced(e)}
```

Comment:
```suggestion
            onClick={setShowAdvanced}
```
And further occurrences